### PR TITLE
Scripts to sync po/pot files from/to transifex

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: sync-main-transifex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+git checkout master
+git checkout -b sync-main-transifex
```
Comment (raveit65) [inline sync-main-transifex]: Why a new branch which needs to be merged after all?

File: sync-main-transifex [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+./autogen.sh --prefix=/usr
+
+rm po/*.po
+tx pull -r MATE.master--mate-utils --minimum-perc 10 --no-interactive -a -f
```
Comment (raveit65) [inline sync-main-transifex]: `minimum_perc = 2` is already set in .tx/config.
Why overwriting existing setting which we use in all repos?

File: sync-main-transifex [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+git checkout master
+git checkout -b sync-main-transifex
+
+./autogen.sh --prefix=/usr
+
+rm po/*.po
```
Comment (raveit65) [inline sync-main-transifex]: Why is this needed?
A pull writes new translations in existing po files.

**Comment by raveit65** [comment]
Why you don`t want to sync /help (user guide) folders?

**Comment by raveit65** [comment]
Normally i run
1. tx pull -af (for /po and /help)
2. committing all changes.
3. copying whole /mate-utils folder to /mate-utils-dist
4. In /mate-utils-dist i run `./autogen.sh && make && make distcheck` to check that we have no merge issues of .po files.
5. If everything is fine --> `git push` to master or 1.22 branch.

**Comment by rbuj** [comment]
It should prevent overwriting the translations on tx, since we only send the catalogs to tx.

File: sync-main-transifex (hunk 3 above)
Comment (rbuj) [reply sync-main-transifex]: It removes unattended locales:
```shell
$ ./sync-transifex-translations
...
$ git diff po/LINGUAS
diff --git a/po/LINGUAS b/po/LINGUAS
index 86a2261..0f775ca 100644
--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -1,5 +1,4 @@
 # please keep this list sorted alphabetically
-#
 af
 am
 ar
@@ -30,15 +29,12 @@ es
 es_AR
 es_CL
 es_CO
-es_ES
 es_MX
-es_PR
 et
 eu
 fa
 fi
 fr
-fr_CA
 frp
 fur
 fy
@@ -59,7 +55,6 @@ ig
 is
 it
 ja
-jv
 ka
 kk
 kn
```

**Comment by raveit65** [comment]
What do you mean with catalogs?
I think it was OK and useful to create the commit `tx: sync with transifex` in the script, but without a push to origin.

**Comment by raveit65** [comment]
> I think it was OK and useful to create the commit tx: sync with transifex in the script, but without a push to origin.

After that the commiter needs to check .po files for this line.
`charset=UTF-8`
We have some repos where the result is `charset=CHARSET` which breaks building.
I have no idea if this is a problem here.
Any way, commit can be edit if something is wrong.

File: sync-main-transifex (hunk 3 above)
Comment (raveit65) [reply sync-main-transifex]: Good, that makes sense.

**Comment by rbuj** [comment]
> After that the commiter needs to check .po files for this line.
> `charset=UTF-8`
> We have some repos where the result is `charset=CHARSET` which breaks building.
> I have no idea if this is a problem here.

done

**Comment by raveit65** [comment]
Syncing .pot files with transifex happens automatic.
We only need to commit changes in .pot file and push the commit to github.
Transifex server sync with .pot files at github 2 times a day.
E.g. https://raw.githubusercontent.com/mate-desktop/mate-utils/master/mate-utils.pot. 
This raw link was added with the `Auto update resources` button at https://www.transifex.com/mate/MATE/content/
Docs at https://docs.transifex.com/projects/updating-content/#automatically-updating-source-files
So i don't think we need the `sync-transifex-pot` script.

But i think it was OK and useful to create the commit tx: sync with transifex with the script, without a push to origin.

**Comment by raveit65** [comment]
@rbuj 
Again, why sync-transifex-pot script?

Syncing .pot files with transifex happens automatic.
We only need to commit changes in .pot file and push the commit to github.
Transifex server sync with .pot files at github 2 times a day.
E.g. https://raw.githubusercontent.com/mate-desktop/mate-utils/master/mate-utils.pot.
This raw link was added with the Auto update resources button at https://www.transifex.com/mate/MATE/content/
Docs at https://docs.transifex.com/projects/updating-content/#automatically-updating-source-files
So i don't think we need the sync-transifex-pot script.

But i think it was OK and useful to create the commit `tx: sync with transifex with the script`, without a push to origin.

**Comment by rbuj** [comment]
> Again, why sync-transifex-pot script?
> 

I have not made any changes yet.
